src/components/actorScriptButton.jsx

```diff
@@ -26,15 +16,11 @@ const ActorScriptButton = ({
         console.error(`error: ${error} when fetching current characters`);
       });
   };
-  // set button value as a string with first and last name
-  const buttonVal = `${firstName} ${lastName}`;
+
   return (
-    <input
-      onClick={onClickChange}
-      className='actorNameButton button-small'
-      type='button'
-      value={buttonVal}
-    />
+    <button onClick={setActorHandler} className="actorNameButton button-small" value={actor.id}>
+      {actor.name}
+    </button>
```

Comment:
This needed to be switched to a button because inputs do not support having a hidden value  and a user visible value.   
This allows us to use the id as the value and use the actor name for the user visible text on the button.